src/main/java/im/status/keycard/KeycardApplet.java

```diff
@@ -1076,12 +1143,31 @@ private void generateKey(APDU apdu) {
       ISOException.throwIt(ISO7816.SW_CONDITIONS_NOT_SATISFIED);
     }
 
-    apduBuffer[ISO7816.OFFSET_LC] = BIP39_SEED_SIZE;
+    // Sanitize P1 input
+    byte exportability = (byte) MASTERSEED_EMPTY;
```

Comment:
You can probably turn this into a util - I've seen the same block twice

Reply:
Yeah, I thought about that. I can change it

Reply:
Actually, I now remember why I didn't make a utility out of this. I want to throw an exception if the wrong value is passed is passed in AND the mapping is not strict for each APDU. Given these, I would rather not bundle into a helper, unless there is a clear advantage for doing so